src/System.IO.Pipelines.Extensions/PipeExtensions.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Sequences;
+
+namespace System.IO.Pipelines
+{
+    public static class PipeExtensions
+    {
+        public static PipeOutput AsOutput(this IPipeWriter writer)
+        {
+            return new PipeOutput(writer);
+        }
+
+        public static PipeOutput AsOutput(this WritableBuffer buffer)
+        {
+            return new PipeOutput(buffer.PipeWriter);
+        }
+
+        public static ISequence<ReadOnlyBuffer<byte>> AsSequence(this ReadableBuffer buffer)
```

Comment:
Why isn't it a method/property on ReadableBuffer?

Reply:
Also, it will box the PipeSequence every time it's called.

Reply:
Because I wanted to trim references and this type is in Extensions assembly. I will fix boxing.

Reply:
ReadableBuffer needs to be parsable without a need for extensions. Otherwise I don't think we can call it a "BCL type"

Reply:
It's neither right now. It shouldn't even be part of pipelines but we haven't consolidated the two types as yet.

Reply:
I think it's ok to merge this, but I think we need to arrive at a set of abstraction that these types will implement as opposed to have extension methods that create adapters.